lib/responder.ts

```diff
@@ -52,14 +54,14 @@ export class Responder {
   //todo: overload this function
   sendAck(timeoutMs: any, responsesRemaining?: any, cb?: any): void {
     if (!cb) {
-      cb = _.last(arguments);
-      if (!_.isFunction(cb)) cb = null;
+      cb = last(arguments);
```

Comment:
Do we need this `last`? This function has 3 parameters, can't you just check if `cb` exists?